src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -301,98 +300,6 @@ public void BindingWorkflow_InstallPackages_Cancellation()
         }
 
         [TestMethod]
```

Comment:
You could [Ignore] the tests and add a comment pointing to the Jira issue for additional file support.